### DISCUSSION TITLE
One Allowed Amount File per EIN?

### DISCUSSION THREAD

**Discussion by Kvillafor** [CMSgov/price-transparency-guide#310] [Q&A] [answered]
@shaselton-usds  With regard to reporting multiple plans in one file: some of our client offers several plans and each plan may have a different plan name, but all the plans have the same EIN and allowed amounts are the same across all plans.  Is this something that could be reported under one plan?

**Comment by CH-PikeSowle** [comment]
@shaselton-usds Our team has the same question.  Thanks

**Comment by shaselton-usds** [comment, marked as answer]
Yes, this can be accomplished by leveraging the table of contents file.
